services/venice-server/src/main/java/com/linkedin/venice/listener/StorageReadRequestsHandler.java

```diff
@@ -403,9 +412,9 @@ private CompletableFuture<ReadResponse> handleMultiGetRequestInParallel(
     AbstractStorageEngine store = getStorageEngine(topic);
```

Comment:
can we rename these to store->storageEngine?

Reply:
Yep, done.